### ISSUE TITLE
Error when running composer install or composer update

### ISSUE THREAD

**Issue by UVLabs** [coenjacobs/mozart#74]
I'm getting the following error when running composer install and not quite sure what's causing it though i know it's coming from mozart's Symfony dependency:

```
composer install
Loading composer repositories with package information
Updating dependencies (including require-dev)
Nothing to install or update
Writing lock file
Generating autoload files
15 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
> Google_Task_Composer::cleanup
Removing 236 google services
> "vendor/bin/mozart" compose

In Finder.php line 589:
                                                                               
  The "/Users/uvdev/Local Sites/rop/app/public/wp-content/plugins/tweet-old-p  
  ost/dependencies/packages/Google_Service_/" directory does not exist.        
                                                                               

compose

Script "vendor/bin/mozart" compose handling the post-update-cmd event returned with error code 1
```

We use Travis for the deployment of our plugin so I do not want to risk pushing this and something goes wrong with the deployment when Travis runs it's `composer install`. 

Here is the branch where I was trying to set up mozart, you can download it and run composer install to see the error: https://github.com/UVLabs/tweet-old-post/tree/issue-672

I think the issue might be because of this command we're running: https://github.com/UVLabs/tweet-old-post/blob/issue-672/composer.json#L38

This is a built-in command by the google client api that lets you delete the google services packages you do not want. In this case we're not using any: https://github.com/googleapis/google-api-php-client#cleaning-up-unused-services

But I did try 

- Removing mozart, 
- Deleting composer lock file
- Adding a google service (_AnalyticsReporting_)
- Then running `composer install` again so files can go back to the initial `vendor` directory, 
- Then I re-added mozart, delete composer.lock file, then run `composer install again`

Mozart goes through the process but gives the same error. From what I can see so far, ~things are working with the plugin~, but i'm not sure how travis might handle this error when it runs it's `composer install` to prepare the package for deploying to wp.org

Update: The abrupt exit actually prevents Mozart/Composer from completing its post command scripts, I noticed the `composer dump-autoload` doesn't run when mozart exits with this error

**Comment by BrianHenryIE** [comment]
`override_autoload` was introduced just for Google API Client! #41. You need to override `apiclient-services`'s too.

```
"override_autoload": {
  "google/apiclient": {
    "classmap": [
      "src/"
    ]
  },
  "google/apiclient-services": {
    "classmap": [
      "src/"
    ]
  }
},
```

But I don't think it's part of 0.5.1, so I changed it to use dev-master. You'll also need the new PR #82 applied to fix the same issue as #58 :

```
"require-dev": {
  "coenjacobs/mozart": "dev-master",
  "cweagans/composer-patches": "~1.0"
}
"extra": {
  "patches": {
    "coenjacobs/mozart": {
      "Replace parent packages over full tree of dependencies": "https://github.com/coenjacobs/mozart/pull/82.patch"
    }
  }
}
```

Then `composer install` runs without issue.

**Comment by coenjacobs** [comment]
PR #82 is now merged into `master` and will soon be in the next beta release of 0.6. This is verified to work when applying that patch, so this can be closed. Please use the current `master` branch to resolve this issue for you, or wait a couple days until the dust has settled and the next beta release is available (final 0.6 shouldn't follow much later).
